public/vendor/plugins/sortt/sortt.js

```diff
@@ -0,0 +1,61 @@
+/*
+  Sortt
+
+  @License MIT
+  @Author 6543
+*/
+
+//creat global function with main routine
+window.sortt=function(normsort,revsort,isdefault){
+  //sortTable [normsort] (revsort) (isdefault)
+
+  //normsort is needet
+  if (typeof normsort  === 'undefined') return false;
+
+  //default values of optinal parameters
+  if (typeof revsort  === 'undefined') revsort = "";
+  if (typeof isdefault  === 'undefined') isdefault = false;
+  if (!(typeof isdefault  === 'boolean')) isdefault = false;
+
+  // parse URL
+  /*  script check if url has already a sort=
+        NO: if sort=“” indikates default (with * prefix) set url_sort=
+        YES: set url_sort variable
+  */
+  url_sort="";
+  if (window.location.search.indexOf("sort=") < 0) {
+    if ( isdefault ) url_sort = normsort;
+  } else {
+    url_sort=window.location.search.substr(window.location.search.indexOf("sort=")+5);
+    if (url_sort.indexOf("&") >= 0) url_sort=url_sort.substr(0,url_sort.indexOf("&"));
+  }
+
+  // generate new URL
+  /*  script check if url_sort and sort attribute is same
+        YES: check if reverse attribute exist -> YES: generate URL with reverse sort param and open
+        NO: generate URL with sort param and open
+  */
+  url_new = window.location.protocol + "//" + window.location.host + window.location.pathname;
+  if (window.location.search.length == 0) {
+    if (url_sort != normsort) {
+      url_new += "?sort=" + normsort
```

Comment:
as additional to my previous comment: `url.searchParams.append("axxx",  "xzxx")` can be used to append parameter

Reply:
url.searchParams works great !
didnt know about it jet :) now i do!